test/test_mps.py

```diff
@@ -7187,6 +7187,244 @@ def test_conv2d_single_stride(self):
             x_gpu = conv_gpu(y_gpu)
             self.assertEqual(x_cpu, x_gpu.cpu(), rtol=1e-03, atol=1e-05)
 
+    def test_grid_sample(self):
```

Comment:
There is no test in TestConsistency ?

Reply:
TestConsistency uses border padding which is unsupported through MPS at this moment.